sovtokenfees/sovtokenfees/test/catchup/test_batch_reverting_for_pool.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+from plenum.test.stasher import delay_rules
+
+from plenum.test.delayers import cDelay
+from sovtokenfees.test.helper import add_fees_request_with_address, check_state
+
+from stp_core.loop.eventually import eventually
+
+from plenum.test.helper import sdk_send_signed_requests, assertExp
+
+from plenum.common.startable import Mode
+
+from plenum.test.node_catchup.helper import ensure_all_nodes_have_same_data
+
+
+def test_revert_works_for_fees_before_catch_up_on_all_nodes(looper, helpers,
```

Comment:
It would be great to have a test when we apply multiple batches (delaying commits), and then start catchup

Reply:
We need such a test for 2 cases:
1) delay Commits on all nodes; apply multiple batches; catch-up
2) delay Commits on one node; apply multiple batches; catch-up